audio/out/ao_sndio.c

```diff
@@ -292,7 +292,7 @@ static void get_state(struct ao *ao, struct mp_pcm_state *state)
     state->delay = p->delay / (double)p->par.rate;
 
     /* report unexpected EOF / underrun */
-    if ((state->queued_samples && state->queued_samples &&
+    if ((state->queued_samples &&
```

Comment:
maybe this meant `&& state->free_samples &&` instead

Reply:
I was thinking about it, but next `state->queued_samples < state->free_samples` covers that, so no difference in practice if we check it or not.